week1/homework/src/server.js

```diff
@@ -9,7 +9,40 @@ function createServer(port) {
   let state = 10;
 
   const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+    const reqUrl = request.url;
+    if (reqUrl === '/' || reqUrl === '/state') {
+      response.writeHead(200, { 'Content-Type': 'application/json' });
+      const home = {
+        'state': state
+      };
+      response.end(JSON.stringify(home, null, 2));
```

Comment:
You don't need the `null` and `2` parameters to stringify.

You can use `response.end()`, but we usually use `response.write()` to write to the response, and only use `response.end()` to indicate the end of the response.